advocacy_docs/partner_docs/FlywayGuide/04-Configuration.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Configuration of Flyway'
+description: 'Walkthrough of Flyway configuration'
+---
+
+### 3.1 Prerequisites
+1. A running EDB Postgres Advanced (EPAS) instance.
+
+2. Flyway Teams Edition installed.
+
+The following step show how to configure Flyway Teams Edition for EDB Postgres Advanced:
+
+Open the flyway.conf file in the flyway-7.11.4/conf directory and add the following EDB Postgres 
+Advanced connection information which will be used by Flyway to connect with EDB Postgres Advanced and perform the migrations.
+
+Following is the description of the configuration options which are set in the flyway.conf.
```

Comment:
remove "following is the description".  
Should be Descriptions of the configuration options that will be set in the flyway.conf file.